### PR TITLE
[A11Y] Fix some Website Accessibility Issues

### LINKED ISSUE #275: [A11Y] Fix Website Accessibility Issues
Fix some issues for Accessibility :

A10Y-Issues_13-3-2024_22656_blackpythondevs com

### PR DESCRIPTION
**Pull request by oleksis**
Fixes #275

### CODE REVIEW

**Comment by kjaymiller** [comment]
Can you include the screenshots with your changes. I think they will better showcase the changes you've Made

**Review by kjaymiller — CHANGES_REQUESTED** [review]
Please add screenshot of affected changes to compare with the original

**Comment by oleksis** [comment]
The changes:

image

File: assets/css/bpd.css
Comment (kjaymiller) [inline assets/css/bpd.css:1]: Why are the changes in the newsletter section when the newsletter wasn't flagged?

**Review by kjaymiller — CHANGES_REQUESTED** [review]

File: assets/css/bpd.css
Comment (oleksis) [reply assets/css/bpd.css:1]: whtat changes?

**Comment by oleksis** [comment]
New changes: 

image

**Comment by oleksis** [comment]
[Add alt attribute specifies an alternate text for an image](https://github.com/BlackPythonDevs/blackpythondevs.github.io/pull/276/commits/ac20d0e9a260e34c176d4d8c22fe77fa3acd714d)

image

**Comment by oleksis** [comment]
The other issues for **A11y** are related to 'pico' CSS vs **white** and **dark** mode for the theme.

File: assets/css/bpd.css
Comment (oleksis) [reply assets/css/bpd.css:1]: when the color for `pico` fix it, we can merge this PR ?

**Review by oleksis — COMMENTED** [review]
look good to me

**Review by kjaymiller — APPROVED** [review]

**Pull request merged**